Monika After Story/game/pong.rpy

```diff
@@ -973,12 +978,12 @@ label mas_pong_dlg_loser:
 
         #Extreme
         else:
-            m 2hub "Great job, [player]!"
-            m 1kua "Ehehe!"
+            m 2hub "You really are something special, [mas_get_player_nickname()]."
+            m 1kua "Every game with you is a thrill."
 
     #Monika loses five times in a row
     elif win_streak_counter == 5:
-        m 2wud "[player]..."
+        m 2wud "[mas_get_player_nickname()]..."
```

Comment:
I think the player's name fits here better than the nickname.

Honestly, I can't see things like 'Love...' or 'My love...' suitable here, you could either exclude these with `exclude_names` keyword argument or let `[player]` be.

Also, the nickname will be in lowercase here, consider using `capitalize` keyword argument.